Dockerfile

```diff
@@ -67,6 +72,7 @@ RUN apt-get update && apt-get install -y \
 	xfsprogs \
 	libzfs-dev \
 	tar \
+	linux-image-`uname -r` \
```

Comment:
@runcom @southerngs I supplied the patch that installs that dependency based on these findings: 

https://github.com/docker/docker/pull/13602#issuecomment-165956483

https://github.com/docker/docker/pull/13602#issuecomment-166323792

In response to @runcom question, it doesn't work on stock ubuntu unless you install this package